pxtblocks/fields/field_gridpicker.ts

```diff
@@ -60,7 +60,7 @@ export class FieldGridPicker extends Blockly.FieldDropdown implements FieldCusto
 
         this.columns_ = parseInt(options.columns) || 4;
         this.maxRows_ = parseInt(options.maxRows) || 0;
-        this.width_ = parseInt(options.width) || 200;
+        this.width_ = parseInt(options.width) || undefined;
```

Comment:
Do you know if there is any scenario where we should have the default be 200 if there isn't a width defined? Is it possible to have the width not be defined but there be elements in the gridpicker and thus introduce weird behavior?